tests/wallet.rs

```diff
@@ -164,22 +182,22 @@ fn send_inscribed_sat() {
 
 #[test]
 fn send_on_mainnnet_refuses_to_work_with_wallet_name_foo() {
-  let rpc_server = test_bitcoincore_rpc::builder().wallet_name("foo").build();
-  let txid = rpc_server.mine_blocks(1)[0].txdata[0].txid();
+  let rpc_server = test_bitcoincore_rpc::builder().build();
 
   CommandBuilder::new(
-    format!("wallet send ord1qcqgs2pps4u4yedfyl5pysdjjncs8et5u8gcumw {txid}:0:0"),
```

Comment:
We should keep this test, and test that send refuses by using rust-bitcoincore-rpc to directly create a wallet with the wrong name.